### ISSUE TITLE
Identity is not created with dfx v0.14.1

### ISSUE THREAD

**Issue by bitdivine** [aviate-labs/setup-dfx#6]
## Expected Behavior
- An identity is created with the supplied key.

## Actual Behavior
- The job prints this many (maybe 1000) times:
```
Please enter a passphrase for your identity: 
```
- Then, when it exits, dfx is installed but not with the supplied key.

## Steps to Reproduce the Problem
- Set credentials in a GitHub workflow
```
      - name: Set credentials
        uses: aviate-labs/setup-dfx@v0.2.3
        with:
          dfx-version: 0.14.1
          dfx-disable-encryption: true
        env:
          DFX_IDENTITY_PEM: ${{ secrets.DFX_IDENTITY_PEM }}
      - name: Verify identity
        run: |
          dfx identity whoami
          dfx identity get-principal          
```
- Observe the github action run.
  - This prints many times:
```
Please enter a passphrase for your identity: 
```
  - Then, verify identity prints  a principal that does not correspond to the key.

Note:  I have tried with and without `dfx-disable-encryption: true`

## Specifications

- Version(s): action version v0.2.3, dfx version 0.14.1

**Comment by q-uint** [comment]
It seems like they deprecated the flag `--disable-encryption` in [v0.13.0](https://github.com/dfinity/sdk/releases/tag/0.13.0). I will create a fix soon @bitdivine.

**Comment by bitdivine** [comment]
Thank you!

**Comment by q-uint** [comment]
Should be fixed in [v0.2.6](https://github.com/aviate-labs/setup-dfx/releases/tag/v0.2.6)!

**Comment by bitdivine** [comment]
Thank you!  I haven't had a chance to test yet but this looks promising.
